docs/src/SUMMARY.md

```diff
@@ -35,6 +35,7 @@
   - [Calling Contracts](./blockchain-development/calling_contracts.md)
 - [Forc](./forc/index.md)
   - [Dependencies](./forc/dependencies.md)
+  - [Plugins](./forc/plugins.md)
```

Comment:
Should we maybe have the "Plugins" chapter come after "Commands" as they are kind of about extending commands?

Reply:
Makes sense!